### DISCUSSION TITLE
add tooltip

### DISCUSSION THREAD

**Discussion by laramaceo** [tnodir/fort#360] [General]
I think it is necessary to have a tooltip in places like this, to be able to read all the content of the menu.

![cuadro 1](https://github.com/user-attachments/assets/f798437c-447b-448d-b743-e2aca59a9a90)

![cuadro 2](https://github.com/user-attachments/assets/5f8326ee-4bff-400c-80e4-2dd95d978f7e)

**Comment by laramaceo** [comment]
There are places where the text is abbreviated and it is not possible to read all the information.

**Comment by tnodir** [comment]
Fixed in [v3.15.3](https://github.com/tnodir/fort/releases/tag/v3.15.3).

**Reply by laramaceo** [reply]
👏   You are unstoppable .
